torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2476,6 +2478,96 @@ Error PyTorchModelLoader::loadListConstruct(const torch::jit::Node *ptNode) {
   RETURN_ERR(addValueMapping(outputs[0], std::move(glowIVal)));
 }
 
+Error PyTorchModelLoader::loadListUnpack(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+
```

Comment:
RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 1, outputs, -1));
Please check this before you call inputs[0].

Reply:
Added in newest commit